eZ/Publish/Core/MVC/Symfony/Matcher/ContentBased/Identifier/ContentType.php

```diff
@@ -52,14 +53,18 @@ public function matchContentInfo(ContentInfo $contentInfo)
 
     public function match(View $view)
     {
-        if (!$view instanceof ContentValueView) {
+        if ($view instanceof ContentTypeView) {
+            $contentTypeIdentifier = $this->repository
+                ->getContentTypeService()
+                ->loadContentType($view->getContentTypeId())->identifier;
```

Comment:
side note, not much to do about it for now: *hmm, these are probably rather expensive, contentType takes long time to build.*